sabnzbd/dirscanner.py

```diff
@@ -124,90 +119,116 @@ def run(self):
             if self.dirscan_speed and not self.shutdown:
                 self.scan()
 
-    def scan(self):
-        """Do one scan of the watched folder"""
+    def __listfiles(self, folder, catdir=None):
```

Comment:
We don't really use the `__` prefix within SABnzbd. I know that it implies privateness in Python, but I'd like to avoid it.
Just a small nitpick.